### ISSUE TITLE
ncu doesn't works when using yarn v2

### ISSUE THREAD

**Issue by andywampir** [raineorshine/npm-check-updates#873]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates` (`11.5.1`)
- [x] I am using `node >= 10.17` (`12.22.1`)

---------------------------

## Steps to Reproduce
1. Install `yarn` v2: `yarn set version berry`
2. Run `ncu`

## Current Behavior
Throws error:
![image](https://user-images.githubusercontent.com/24408703/115852376-73f1fb80-a441-11eb-9a29-9df3886ff3c6.png)

## Expected Behavior
Work well as when using `--packageManager npm`

**Comment by raineorshine** [comment]
@MarxZhou @Tchoupinax Any interest in helping with this?

**Comment by Tchoupinax** [comment]
Hello,

I can't reproduce the issue on my system. However, I'm on Linux on this Desktop and I see you are using Windows. Maybe an issue about Windows ?

How did you install `ncu` ?
It would be nice if you can reproduce the error within a Dockerfile ^^

**Comment by ratson** [comment]
@Tchoupinax You can check the bug with
```sh
git clone -b npm-check-updates/yarn2 https://github.com/ratson/bug-report.git
cd bug-report
yarn
yarn ncu
```

**Comment by Tchoupinax** [comment]
Thanks @ratson for the example.
I found where the error is thrown, it was not easy because the thrown error is empty. I have no idea why this line throws, but the catch allows the process to correctly continue.

@raineorshine According to you, is this catch enought or you want to add a specific process ?

**Comment by raineorshine** [comment]
An error is thrown by `yarn global dir`. The question is, why does `yarn global dir` work with yarn v1 and not with yarn v2? We need to know the cause before we can make the appropriate fix.

**Comment by ratson** [comment]
@Tchoupinax @raineorshine Here is the error.
```sh-session
❯ yarn global dir
Usage Error: The 'yarn global' commands have been removed in 2.x - consider using 'yarn dlx' or a third-party plugin instead

$ yarn run [--inspect] [--inspect-brk] <scriptName> ...
```
So return `null` for such case would be fine.

**Comment by raineorshine** [comment]
Fixed in 0628a0e212fe2e305934cb0a56d2b2f76faee23d and published in `v11.5.6`.
